x-pack/platform/plugins/shared/cases/public/components/all_cases/all_cases_list.test.tsx

```diff
@@ -90,6 +90,44 @@ const mockKibana = () => {
   } as unknown as ReturnType<typeof useKibana>);
 };
 
+// eslint-disable-next-line prefer-object-spread
```

Comment:
changed this initially, not needed anymore but keeping just because it's cleaner/easier to reuse if needed